cmd/buildah/common.go

```diff
@@ -133,3 +133,11 @@ func defaultFormat() string {
 	}
 	return "oci"
 }
+
+func useLayers() bool {
+	layers := os.Getenv("BUILDAH_LAYERS")
+	if layers == "true" {
```

Comment:
Add 
strings.ToLower(layers) and check also against "1"

Reply:
Fixed